subsys/net/l2/ethernet/gptp/gptp_md.c

```diff
@@ -583,6 +583,12 @@ static void gptp_md_pdelay_req_state_machine(int port)
 		state->state = GPTP_PDELAY_REQ_NOT_ENABLED;
```

Comment:
You are mixing core gptp change with this sample change. Please create a separate commit for the core gptp change.